ccd-definition/CaseEventToComplexTypes/create-order/create-order-gatekeeping.json

```diff
@@ -1,29 +1,29 @@
 [
   {
     "LiveFrom": "01/01/2017",
-    "ID": "C21OrderComplexType",
-    "CaseEventID": "createC21OrderGatekeeping",
-    "CaseFieldID": "c21Order",
+    "ID": "GeneratedOrderComplexType",
```

Comment:
Generated order is an order in a submitted state right? Why not `SubmittedOrder`?

Reply:
It's not strictly an order in a submitted state: the title/details get entered and stored in this object on the first page, then in mid event the object has a document added to it, and then after submission the date and judge are added to it. Ideally it's just an 'Order' but I can't use that model - regardless I do think SubmittedOrder sounds better than GeneratedOrder, so I might go ahead with that name change, although since we have a 'Submitted' case state, it might be confusing? I'm not sure, naming this is really hard because 'Order' and 'Orders' are already taken

Reply:
Well in that case should we not have `temporaryOrder` and then just `Order`?

Reply:
I think SubmittedOrder could be a bit confusing if it is as Joe said it's not strictly an order in a submitted state. I think lets stick with GeneratedOrder for now as @akomisarek is happy with this and we can refactor this again in the future if we see fit. Happy with this @Jamesnelson117 ?

Reply:
going to discuss this with @akomisarek

Reply:
Im happy with any name really, ideally the best name is 'Order' but we can't use it. And we definitely are not using Order2 or 0rder 😄 GeneratedOrder is good enough